tests/optional_kernel_features/kernel_features_common.h

```diff
@@ -126,12 +124,11 @@ inline void dummy_function_non_decorated(const sycl::accessor<bool, 1> &acc) {
 /**
  * @brief Macro for generating code that will use TYPE
  */
-#define USE_FEATURE(TYPE)       \
-  unsigned long long temp = 42; \
-  TYPE feature1(temp);          \
-  TYPE feature2(temp);          \
-  feature1 += 42;               \
-  acc[0] = (feature1 == feature2);
+#define USE_FEATURE(TYPE)           \
+  unsigned long long temp = acc[0]; \
+  TYPE feature(temp);               \
+  feature += 1;                     \
+  acc[0] = feature * 4 / 3 > 2;
```

Comment:
Just curious while you are here. Since this is similar to the other 2 usages, why this macro is not used everywhere?

Reply:
Applied this change